src/main/java/org/opentripplanner/openstreetmap/tagmapping/DefaultMapper.java

```diff
@@ -569,7 +569,8 @@ public void populateProperties(WayPropertySet props) {
     props.setMixinProperties("surface=wood", ofBicycleSafety(1.18));
 
     props.setMixinProperties("surface=cobblestone", ofBicycleSafety(1.3));
-    props.setMixinProperties("surface=cobblestone:flattened", ofBicycleSafety(1.3));
+    props.setMixinProperties("surface=sett", ofBicycleSafety(1.3));
+    props.setMixinProperties("surface=unhewn_cobblestone", ofBicycleSafety(1.5));
```

Comment:
I think this might have an even higher safety value

Reply:
I agree. But how much? I thought that 100, like surface=sand, is a slight exaggeration.